test/index-ejs.spec.js

```diff
@@ -36,7 +37,7 @@ describe('index.ejs template', () => {
             skyux: skyPagesConfig.skyux
           }),
           function () {
-            this.plugin('emit', (compilation) => {
+            this.hooks.emit.tap('index-ejs.spec.js-plugin', (compilation) => {
```

Comment:
These changes may not be necessarily, but I was consistently getting timeouts on the first run of this test locally.